frontend/src/components/GameplayScreen/GameplayScreen.jsx

```diff
@@ -12,18 +12,21 @@ import LoadingSpinner from "../LoadingPage/LoadingSpinner"
 export default function GameplayScreen({positions ,setPositions, latitude, setLatitude, longitude, setLongitude, country_id}) {
   const [guessed,setGuessed] = useState(false)
   const {cities, setCities} = useCitiesContext();
-  var [categorizedCities, setCategorizedCities] = useState([])
+  var categorizedCities = [];
   const[error, setError] = useState("")
   const [isFetching, setIsFetching] = useState(false)
   const [data, setData] = useState([]);
 
-  console.log(cities)
-  console.log(country_id)
-  // for(the size of cities array){
-  // if(cities[i].country_id == country_id){
-  // categorizedCities.push(cities[i])
-  // }
-  // }
+  //console.log(cities)
+  //console.log(country_id)
```

Comment:
RECOMMENDED: Use Javascript Console Utils (VSCode Extension) for creating and removing log statements efficiently.